packages/plugins/rubygem-redhat_access/rubygem-redhat_access.spec

```diff
@@ -0,0 +1,131 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name redhat_access
+
+%global foreman_dir /usr/share/foreman
+%global foreman_bundlerd_dir %foreman_dir/bundler.d
+#%global foreman_assets_dir %foreman_dir/public/assets
+%global rubygem_redhat_access_dir %{gem_dir}/gems/%{gem_name}-%{version}
+%global gem_spec %{gem_dir}/specifications/%{gem_name}-%{version}.gemspec
+%global gem_cache %{gem_dir}/cache/%{gem_name}-%{version}.gem
+
+
+
+%global scl_rake /usr/bin/%{?scl:%{scl_prefix}}rake
+
+Name: %{?scl_prefix}rubygem-foreman-%{gem_name}
+Version: 2.1.0
+Release: 1%{?dist}
+Summary: Foreman engine to access Red Hat knowledge base and manage support cases.
+Group: Development/Languages
+License: GPLv2+
```

Comment:
The gemspec doesn't specify it, but the license file states GPLv3. https://github.com/redhataccess/foreman-plugin/pull/19 should make it explicit.